typescript/sdk/src/hook/read.ts

```diff
@@ -115,8 +119,8 @@ export class EvmHookReader implements HookReader {
   async deriveMerkleTreeConfig(
     address: Address,
   ): Promise<WithAddress<MerkleTreeHookConfig>> {
-    const hook = MerkleTreeHook__factory.connect(address, this.provider);
-    assert((await hook.hookType()) === OnchainHookType.MERKLE_TREE);
```

Comment:
Just noticed this file imports the wrong `assert` fn. For cross-compat, it should be importing our own from utils.